### ISSUE TITLE
Post-condition for contracts wrong

### ISSUE THREAD

**Issue by friedger** [leather-io/extension#1357]
The following post condition
```
 makeContractSTXPostCondition(
            address,
            name,
            FungibleConditionCode.GreaterEqual,
            new BN(0)
          )
```
resulted in a post condition for `address` instead of `address.name`

See

https://explorer.stacks.co/txid/0x18bdc2b484566fdd22b15e26fdacb5cd8a7c200e65342dd6061caba87ceb11e0?chain=testnet (wrong via extension)

https://explorer.stacks.co/txid/0x8f89a9a186df36aa4ac56a5bc601dacf34b0a66d8648baa7e98610179656a3e8?chain=testnet (correct via @stacks/transactions)

**Comment by aulneau** [comment]
Can you rephrase? it sounds like this is expected

**Comment by friedger** [comment]
I expect the pc for the contract principal, not for the standard principal (updated the issue)

**Comment by markmhendrickson** [comment]
@aulneau I believe you mentioned during the call today that we need to investigate whether there's a collision somewhere in how address values are handled?

**Comment by markmhendrickson** [comment]
This should be fixed now. @friedger please reopen if you're still having trouble here.

**Comment by friedger** [comment]
Thank you
